tools/scripts/stm32.mk

```diff
@@ -140,6 +142,10 @@ CC = arm-none-eabi-gcc
 GDB = arm-none-eabi-gdb
 GDB_PORT = 50000
 OC = arm-none-eabi-objcopy
+ifeq (,$(COMPILER_BIN))
+COMPILER_BIN = $(realpath $(dir $(call rwildcard, $(STM32CUBEIDE)/plugins, *arm-none-eabi-gcc)))
+endif
+export PATH := $(COMPILER_BIN):$(PATH)
```

Comment:
are these changes related to this commit or should they be moved to the third commit?

Reply:
This was reverting a previous accidental removal, will change this now

Reply:
This has not been fixed (simply remove this change from this commit)